Utils/layouts/banner.js

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import { BannerSection, SubSection, BannerText } from "."
+import { i18n, withTranslation } from "../translation/i18n";
+import {Col, Heading, resolveMedia} from "@moneypensionservice/directories";
+import styled from "styled-components";
+
+import LogoEN from "./LogoEN.jsx"
+import LogoCY from "./LogoCY.jsx"
+
+export const MoneyHelperLogo = styled(Col)`
+  display: flex;
```

Comment:
I don't think applying `display: flex` here makes any difference, if anything that should be applied to the parent element, though in fact the layout requires this element to float such that it doesn't appear below the text at some widths.

Reply:
Ooops! Forgot to remove that